bibliography.bib

```diff
@@ -8224,3 +8224,612 @@ @InProceedings {ChiKimStaKim2021humanoids
+    year         = {2015},
+    pages        = {529--533},
+    doi          = {10.1038/nature14236},
+    note         = {},
+    annote       = {},
+}
+
+@Article{Nob2006NATUBIOT,
+    title        = {What is a support vector machine?},
+    author       = Noble_WS,
+    journal      = NATUBIOT,
+    series       = {},
+    volume       = {24},
+    number       = {12},
+    year         = {2006},
+    pages        = {1565--1567},
+    doi          = {10.1038/nbt1206-1565},
+    note         = {},
+    annote       = {},
+}
+
+@Article{HorStiWhi1989NN,
+    title        = {Multilayer feedforward networks are universal approximators},
+    author       = Hornik_K # and # Stinchcombe_M # and # White_H,
+    journal      = NN,
+    series       = {},
+    volume       = {2},
+    number       = {5},
+    year         = {1989},
+    pages        = {359--366},
+    doi          = {10.1016/0893-6080(89)90020-8},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BelHsuMaMan2019PNAS,
+    title        = {Reconciling modern machine-learning practice and the classical bias--variance trade-off},
+    author       = Belkin_M # and # Hsu_D # and # Ma_S # and # Mandal_S,
+    journal      = PNAS,
+    series       = {},
+    volume       = {116},
+    number       = {32},
+    year         = {2019},
+    pages        = {15849--15854},
+    doi          = {10.1073/pnas.1903070116},
+    note         = {},
+    annote       = {},
+}
+
+@Article{TriNol2011AL,
+    title        = {Engineering the evolution of self-organizing behaviors in swarm robotics: A case study},
+    author       = Trianni_V # and # Nolfi_S,
+    journal      = AL,
+    series       = {},
+    volume       = {17},
+    number       = {3},
+    year         = {2011},
+    pages        = {183--202},
+    doi          = {10.1162/artl_a_00031},
+    note         = {},
+    annote       = {},
+}
+
+@InCollection {NgCoaDie-etal2006iser,
+    title        = {Autonomous inverted helicopter flight via reinforcement learning},
+    author       = Ng_AY # and # Coates_A # and # Diel_M # and # Ganapathi_V # and # Schulte_J # and # Tse_B # and # Berger_E # and # Liang_E,
+    booktitle    = iser_2006,
+    editor       = Ang_MH # and # Khatib_O,
+    series       = {},
+    volume       = {21},
+    number       = {},
+    year         = {2006},
+    pages        = {363--372},
+    publisher    = pub_SP,
+    address      = Berlin_DEU,
+    doi          = {10.1007/11552246_35},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {BusDeSBab2006icarcv,
+    title        = {Decentralized reinforcement learning control of a robotic manipulator},
+    author       = Busoniu_L # and # DeSchutter_B # and # Babuska_R,
+    booktitle    = icarcv_2006,
+    editor       = {},
+    series       = {},
+    volume       = {1319},
+    number       = {},
+    year         = {2006},
+    pages        = {1--6},
+    publisher    = pub_IEEE,
+    address      = {},
```

Comment:
Please add address to the entries being published by IEEE. IEEE is normally located in Piscataway but check title pages, etc. of conference proceedings. Also applies to later entries.